README.md

```diff
@@ -55,4 +55,6 @@ This function will 'save' all the strings sent to it in any format, be it a list
 
 **Example**: [cat.js](examples/cat.js)
 
-**Notes**: When `typeof` of an input evaluates to `object` or `undefined`, this argument will be ignored. This happens to any kind of object, map, null or undefined values. To know more about the `typeof` operand, follow this link: [typeof - MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof)
+**Notes**:
+- When `typeof` of an input evaluates to `object` or `undefined`, this argument will be ignored. This happens to any kind of object, map, null or undefined values. To know more about the `typeof` operand, follow this link: [typeof - MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof)
+- While arrow functions maybe more clean and concise, the do not bind the `this` object, so in case you need to use the helper inside a function that is going to be passed as an argument, write an old fashion function. More info on arrow functions: [Arrow Functions - MDN](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
```

Comment:
yeah, a typo... and tried to fix a little the way I redacted the explanation. What do you think @ryanez? Commit 7a89758